### PR TITLE
feat: change name field disabled to enabled

### LINKED ISSUE #1324: Change name field disabled to enabled

### PR DESCRIPTION
**Pull request by uziasferreirazup**
### Related Issues

Closes #1324 

### Description and Example

Adjust name field disabled to enabled

### Checklist

Please, check if these important points are met using `[x]`:

- [X] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [X] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [X] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

**Comment by botbeaglezup** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: dlopen(/Users/runner/.gem/ruby/2.7.0/gems/ox-2.14.0/ext/ox/ox.bundle, 9): Library not loaded: /usr/local/opt/ruby/lib/libruby.2.7.dylib
  Referenced from: /Users/runner/.gem/ruby/2.7.0/gems/ox-2.14.0/ext/ox/ox.bundle
  Reason: image not found - /Users/runner/.gem/ruby/2.7.0/gems/ox-2.14.0/ext/ox/ox.bundle. Updating the Danger gem might fix the issue. Your Danger version: 8.2.1, latest Danger version: 8.2.2

```
 #  from danger/backend/Dangerfile:6
 #  -------------------------------------------
 #    checkstyle_format.base_path = Dir.pwd
 >    checkstyle_format.report file_name
 #  end
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_backend">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: android/beagle/src/test/java/br/com/zup/beagle/android/components/ButtonTest.kt [hunk 1]
```diff
@@ -118,12 +118,12 @@ internal class ButtonTest : BaseComponentTest() {
         }
     }
 
-    @DisplayName("When passing isEnabled")
+    @DisplayName("When set enabled")
```
Comment (igorgiumellizup) [inline android/beagle/src/test/java/br/com/zup/beagle/android/components/ButtonTest.kt:121]: Which one it represents?
When set enabled true?
When set enabled false?

File: backend/widgets-dsl/src/main/kotlin/br/com/zup/beagle/ui/ButtonBuilder.kt [hunk 2]
```diff
@@ -32,17 +32,22 @@ class ButtonBuilder : BeagleBuilder<Button> {
...
     var clickAnalyticsEvent: ClickEvent? = null
+    var enabled: Bind<Boolean>? = null
 
     fun text(text: Bind<String>) = this.apply { this.text = text }
+    fun enabled(enabled: Bind<Boolean>) = this.apply { this.enabled = enabled }
     fun styleId(styleId: String?) = this.apply { this.styleId = styleId }
     fun onPress(onPress: List<Action>?) = this.apply { this.onPress = onPress?.toMutableList() }
-    fun clickAnalyticsEvent(clickAnalyticsEvent: ClickEvent?)
-        = this.apply { this.clickAnalyticsEvent = clickAnalyticsEvent }
+    fun clickAnalyticsEvent(clickAnalyticsEvent: ClickEvent?) = this.apply { this.clickAnalyticsEvent = clickAnalyticsEvent }
```
Comment (botbeaglezup) [inline backend/widgets-dsl/src/main/kotlin/br/com/zup/beagle/ui/ButtonBuilder.kt:41]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/d2691632bab6fdfb63f5a386ae49858d4f5ea532/backend/widgets-dsl/src/main/kotlin/br/com/zup/beagle/ui/ButtonBuilder.kt#L41"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>

File: android/beagle/src/test/java/br/com/zup/beagle/android/components/ButtonTest.kt (hunk 1 above)
Comment (uziasferreirazup) [reply android/beagle/src/test/java/br/com/zup/beagle/android/components/ButtonTest.kt:121]: this is defined by then

**Comment by botbeaglezup** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: dlopen(/Users/runner/.gem/ruby/2.7.0/gems/ox-2.14.0/ext/ox/ox.bundle, 9): Library not loaded: /usr/local/opt/ruby/lib/libruby.2.7.dylib
  Referenced from: /Users/runner/.gem/ruby/2.7.0/gems/ox-2.14.0/ext/ox/ox.bundle
  Reason: image not found - /Users/runner/.gem/ruby/2.7.0/gems/ox-2.14.0/ext/ox/ox.bundle. Updating the Danger gem might fix the issue. Your Danger version: 8.2.1, latest Danger version: 8.2.2

```
 #  from danger/android/Dangerfile:6
 #  -------------------------------------------
 #    checkstyle_format.base_path = Dir.pwd
 >    checkstyle_format.report file_name
 #  end
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by matheusribeirozup** [comment]
Please, change component in Flutter as well.

**Review by theffc — APPROVED** [review]

**Review by matheusribeirozup — APPROVED** [review]

**Review by paulomeurerzup — APPROVED** [review]

**Review by theffc — APPROVED** [review]

**Pull request merged**